config/locales/en.yml

```
@@ -3941,6 +3944,8 @@ See the %{link} to find out more about %{sitename}'s features and to start using
           cancel_orders: "Cancel Orders"
           resend_confirmation: "Resend Confirmation"
           resend_confirmation_confirm_html: "This will resend the confirmation email to the customer.<br />Are you sure you want to proceed?"
+          send_invoice: "Send Invoices"
+          send_invoice_confirm_html: "This will send the invoice to the customer.<br />Are you sure you want to proceed?"
```

A few thoughts:

I'm tempted to suggest we use `one` and `other` because there could be multiple customers. But let's not make life hard ;).
Instead, it could simply be worded differently to support both, like  "This will send customer invoices for all selected orders..."

The invoices are going to be emailed, so I'd use the word "email".

_update_: The invoices are only emailed if completed/resumed, so we can reveal that too.

One more thing, in HTML5, the slash is no longer required in a void tag, so a line break can simply be `<br>`. I always add a space after the full-stop, in case the html ever gets stripped.

Therefore I suggest:
```suggestion
          send_invoice_confirm_html: "This will email customer invoices for all selected complete orders. <br>Are you sure you want to proceed?"
```

---

I agree, thanks for english correction!